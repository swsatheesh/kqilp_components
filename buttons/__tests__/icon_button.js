import React from 'react';
import renderer from 'react-test-renderer';
import IconButton from '../icon_button';

jest.unmock('../icon_button');

describe('component', () => {
  describe('<IconButton />', () => {
    it('should render correctly', () => {
      const button = renderer.create(
        <IconButton iconName={'home'} />
      ).toJSON();
      expect(button).toMatchSnapshot();
    });
  });
});
