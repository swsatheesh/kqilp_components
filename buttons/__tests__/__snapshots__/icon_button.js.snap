// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <IconButton /> should render correctly 1`] = `
<button
  className="btn icon-button"
  disabled={false}
  onClick={undefined}
>
  <span
    className="fa fa-home"
  />
  
</button>
`;
